stripe.php

```diff
@@ -112,6 +112,28 @@ function stripe_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
 }
 
 /**
+ * Implementation of hook_civicrm_validateForm().
+ *
+ * Prevent server validation of cc fields
+ *
+ * @param $formName - the name of the form
+ * @param $fields - Array of name value pairs for all 'POST'ed form values
+ * @param $files - Array of file properties as sent by PHP POST protocol
+ * @param $form - reference to the form object
+ * @param $errors - Reference to the errors array.
+ */
+function stripe_civicrm_validateForm($formName, &$fields, &$files, &$form, &$errors) {
+  if (isset($form->_paymentProcessor['payment_processor_type']) &&$form->_paymentProcessor['payment_processor_type'] == 'Stripe') { 
```

Comment:
I'm having issues with this as well. It seems that the `$form` object doesn't always have this information. Actually, in my testing, all of the backend payment forms I've tested so far don't seem to store the processor type here:

![image](https://cloud.githubusercontent.com/assets/1479206/6031960/9e7344a0-abd1-11e4-911c-029d831cfe0c.png)

The result is that the validation doesn't get stripped and the contribution fails. 

How about looking at `$form->_submitValues['payment_processor_id']` instead? I'm not sure how to get the extension's ID but i'm sure there's a way... @drastik?